collect_app/src/main/java/org/odk/collect/android/widgets/GeoPointWidget.java

```diff
@@ -60,101 +55,71 @@
+                draggable = false;
+                useMap = true;
+            }
         }
 
         readOnly = prompt.isReadOnly();
-
         answerDisplay = getCenteredAnswerTextView();
 
-        viewButton = getSimpleButton(getContext().getString(R.string.get_point), R.id.get_point);
-
         getLocationButton = getSimpleButton(R.id.get_location);
 
         // finish complex layout
-        // control what gets shown with setVisibility(View.GONE)
         LinearLayout answerLayout = new LinearLayout(getContext());
         answerLayout.setOrientation(LinearLayout.VERTICAL);
         answerLayout.addView(getLocationButton);
-        answerLayout.addView(viewButton);
         answerLayout.addView(answerDisplay);
         addAnswerView(answerLayout);
 
         // Set vars Label/text for button enable view or collect...
         boolean dataAvailable = false;
-        String s = prompt.getAnswerText();
-        if (s != null && !s.equals("")) {
+        String answer = prompt.getAnswerText();
+        if (answer != null && !answer.equals("")) {
             dataAvailable = true;
-            setBinaryData(s);
+            setBinaryData(answer);
         }
         updateButtonLabelsAndVisibility(dataAvailable);
-
     }
 
     private void updateButtonLabelsAndVisibility(boolean dataAvailable) {
-        // BUT for mapsV2, we only show the getLocationButton, altering its text.
-        // for maps, we show the view button.
-
-        if (useMapsV2 && useMaps) {
-            // hide the view button
-            viewButton.setVisibility(View.GONE);
+        if (useMap) {
             if (readOnly) {
-                //READ_ONLY View
-                getLocationButton.setText(
-                        getContext().getString(R.string.geopoint_view_read_only));
+                getLocationButton.setText(R.string.geopoint_view_read_only);
             } else {
-                if (stringAnswer != null && !stringAnswer.isEmpty()) {
-                    getLocationButton.setText(
-                            getContext().getString(R.string.view_change_location));
-                } else {
-                    getLocationButton.setText(getContext().getString(R.string.get_point));
-                }
+                getLocationButton.setText(
+                    dataAvailable ? R.string.view_change_location : R.string.get_point);
             }
         } else {
             if (!readOnly) {
-                getLocationButton.setText(getContext().getString(
-                        dataAvailable ? R.string.change_location : R.string.get_point));
-            }
-
-            if (useMaps) {
-                // show the view button
-                viewButton.setVisibility(View.VISIBLE);
-                viewButton.setEnabled(dataAvailable);
-            } else {
-                viewButton.setVisibility(View.GONE);
+                getLocationButton.setText(
+                    dataAvailable ? R.string.change_location : R.string.get_point);
```

Comment:
Yup—I removed the viewButton because it's never used.  `useMaps` is only true when `useMapsV2` is true, so  `viewButton.setVisibility(View.VISIBLE)` is never executed.